.github/workflows/macos.yml

```diff
@@ -48,10 +65,18 @@ jobs:
       )}}
 
     steps:
+      - name: Enable Launchable conditionally
+        id: enable_launchable
+        run: echo "enable_launchable=true" >> $GITHUB_OUTPUT
+        working-directory:
+        if: ${{ (github.repository == 'ruby/ruby' || (github.repository != 'ruby/ruby' && env.LAUNCHABLE_TOKEN)) && (matrix.test_task == 'check' || matrix.test_task == 'test-all') }}
```

Comment:
I would suggest refactoring like this. The code was inspired from other parts such as [this part](https://github.com/ruby/ruby/blob/b1431ce427eb193f45fe536f1fdfdf22bbe02e3d/.github/workflows/macos.yml#L42-L48). I hesitate applying the `${{(` and `)}}` without one space like other parts.

```diff
diff --git a/.github/workflows/macos.yml b/.github/workflows/macos.yml
index 75b642e7e2..e27cf0dc7d 100644
--- a/.github/workflows/macos.yml
+++ b/.github/workflows/macos.yml
@@ -69,7 +69,12 @@ jobs:
         id: enable_launchable
         run: echo "enable_launchable=true" >> $GITHUB_OUTPUT
         working-directory:
-        if: ${{ (github.repository == 'ruby/ruby' || (github.repository != 'ruby/ruby' && env.LAUNCHABLE_TOKEN)) && (matrix.test_task == 'check' || matrix.test_task == 'test-all') }}
+        if: >-
+          ${{
+          (github.repository == 'ruby/ruby' ||
+          (github.repository != 'ruby/ruby' && env.LAUNCHABLE_TOKEN)) &&
+          (matrix.test_task == 'check' || matrix.test_task == 'test-all')
+          }}

       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
```

Reply:
Updated